src/helpers/createPost.ts

```diff
@@ -15,21 +16,33 @@ export default async function (
   replyToId?: string
 ) {
   TextFormStore.loading = true
+  TextFormStore.error = null
 
   let currentAccount
   const { privateKey } = BurnerWalletStore
-  const { account } = walletStore
+  let { account } = walletStore
```

Comment:
Why don't you use `useSnapshot` here? It looks a little bit strange in my opinion

Reply:
Because it's callback, any hook won't work inside of it
![image](https://user-images.githubusercontent.com/49658988/196525786-ae5f8632-1ba2-4b6f-a81d-d2343b79beac.png)

Reply:
Right